[2025-02-15T02:14:40.304+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.export_daily_forecast manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T02:14:40.319+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.export_daily_forecast manual__2025-02-15T00:56:13.638531+00:00 [queued]>
[2025-02-15T02:14:40.320+0000] {taskinstance.py:1331} INFO - Starting attempt 7 of 7
[2025-02-15T02:14:40.341+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): export_daily_forecast> on 2025-02-15 00:56:13.638531+00:00
[2025-02-15T02:14:40.347+0000] {standard_task_runner.py:57} INFO - Started process 442 to run task
[2025-02-15T02:14:40.356+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'export_daily_forecast', 'manual__2025-02-15T00:56:13.638531+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpssf5n943']
[2025-02-15T02:14:40.359+0000] {standard_task_runner.py:85} INFO - Job 102: Subtask export_daily_forecast
[2025-02-15T02:14:40.484+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.export_daily_forecast manual__2025-02-15T00:56:13.638531+00:00 [running]> on host cbea98f91475
[2025-02-15T02:14:40.664+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='export_daily_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-02-15T00:56:13.638531+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-15T00:56:13.638531+00:00'
[2025-02-15T02:14:40.713+0000] {logging_mixin.py:149} INFO - Index(['dt', 'summary', 'pressure', 'humidity', 'dew_point', 'wind_speed',
       'wind_gust', 'clouds', 'pop', 'uvi', 'rain', 'snow', 'main',
       'description', 'min_temp', 'max_temp', 'day_temp', 'day_feels_like',
       'location_id', 'current_weather_id'],
      dtype='object')
[2025-02-15T02:14:40.746+0000] {logging_mixin.py:149} INFO - Data inserted successfully!
[2025-02-15T02:14:40.748+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-15T02:14:40.779+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=export_daily_forecast, execution_date=20250215T005613, start_date=20250215T021440, end_date=20250215T021440
[2025-02-15T02:14:40.818+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-15T02:14:40.866+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
